data.js

```diff
@@ -0,0 +1,48 @@
+const data = {
+  features: [
+    {
+      key: 0,
+      featureImage: './img/b1.jpeg',
+      featureName: 'Business',
+      featureDescription: ' "They manage the knowledge transfer really well."',
+      featureMessage: ' Because they build a relationship that is based on trust . Over they years they have proven that delivery is fast and stable',
+    },
+    {
+      key: 1,
+      featureImage: './img/b2.jpeg',
+      featureName: 'Business',
+      featureDescription: '"They manage the knowledge transfer really well."',
+      featureMessage: ' Because they build a relationship that is based on trust . Over they years they have proven that delivery is fast and stable',
+    },
+    {
+      key: 2,
+      featureImage: './img/b7.jpeg',
+      featureName: 'Business',
```

Comment:
- [ ] Good job dynamically rendering this section of the page and using the right semantic HTML tags to group them. However, there are a few styling issues to consider below:
- The text and image should be horizontally aligned.
- The title should be big and bold.
- Reduce the spacing between the header and the section items.
- The subtitle should be coloured orange and italicised.
- The image should have a decorative shadow at the top left corner.
- Increase the width and height of the image.
- Remove the rounded corners in this section of the page.

_See images below._

**Original** | **Your project**
  ----------- | -----------
  ![image](https://user-images.githubusercontent.com/49621279/123960757-88340680-d964-11eb-9bc0-03bb416e3ba1.png) | ![image](https://user-images.githubusercontent.com/49621279/123980501-7196ab00-d976-11eb-8fce-6fb9baf19558.png)